### PR TITLE
Added conversation update activity handler

### PR DESCRIPTION
**Pull request by coupa-sbawaskar**

### CODE REVIEW

**Comment by PrasadG193** [comment]
Hey @coupa-sbawaskar ,
Thanks for the PR. The CI is failing due to go lint checks. Could you please fix that?

File: core/activity/handler.go [hunk 1]
```diff
@@ -30,14 +30,16 @@ import (
...
 
 // HandlerFuncs is an adaptor to let client program specify as many or
 // as few functions to handle events of the connector service while still implementing
 // Handler.
 type HandlerFuncs struct {
-	OnMessageFunc func(turn *TurnContext) (schema.Activity, error)
-	OnInvokeFunc  func(turn *TurnContext) (schema.Activity, error)
+	OnMessageFunc            func(turn *TurnContext) (schema.Activity, error)
+	OnInvokeFunc             func(turn *TurnContext) (schema.Activity, error)
+	OnConversationUpdateFunc func(context *TurnContext) (schema.Activity, error)
```
Comment (PrasadG193) [inline core/activity/handler.go:42]:
```suggestion
	OnConversationUpdateFunc func(turn *TurnContext) (schema.Activity, error)
```
minor: let's follow the uniform pattern

**Comment by PrasadG193** [comment]
LGTM. Thanks for the PR @coupa-sbawaskar

**Comment by PrasadG193** [comment]
@coupa-sbawaskar Can you please raise the PR against develop? `develop` is the default branch. `master` points to the latest stable ralease

**Comment by PrasadG193** [comment]
I had to revert the PR merge. Could you please raise the PR against develop again?

**Comment by coupa-sbawaskar** [comment]
> I had to revert the PR merge. Could you please raise the PR against develop again?

sure

**Comment by PrasadG193** [comment]
Hi @coupa-sbawaskar,
We will be cutting the branch for a minor release. It would be great if we can get this in for the next release. Will it be possible for you to raise the PR?

**Pull request merged**